test/WebSites/ModelBindingWebSite/Controllers/VehicleController.cs

```diff
@@ -13,6 +14,21 @@ namespace ModelBindingWebSite
 {
     public class VehicleController : Controller
     {
+        private static Dictionary<int, VehicleViewModel> _vehicles = new Dictionary<int, VehicleViewModel>
+        {
+            {
+                42,
+                new VehicleViewModel
+                {
+                    InspectedDates = new[] { DateTimeOffset.Parse("1 April 2001"), },
```

Comment:
that would only make explicit use of the current culture.  are you thinking that's necessary?  or do you have another culture in mind?